runtime/doc/eval.txt

```diff
@@ -4709,7 +4723,7 @@ inputdialog({prompt} [, {text} [, {cancelreturn}]])		*inputdialog()*
 		omitted an empty string is returned.
 		Hitting <Enter> works like pressing the OK button.  Hitting
 		<Esc> works like pressing the Cancel button.
-		NOTE: Command-line completion is not supported.
+		NOTE: Command-line completion is not supported in the GUI.
```

Comment:
That line could be removed, it's up to the GUI to decide that. And until we externalize dialogs, GUIs have no choice but to use the TUI layout--so it would be supported there. If I'm not mistaken...